### PR TITLE
update configuring sriov doc

### PR DESCRIPTION
**Pull request by zenghui-shi**

### CODE REVIEW

**Comment by jboxman** [comment]
Hi @zshi-redhat, is this good? For what versions? Thanks!

**Comment by zshi-redhat** [comment]
This is created against openshift-doc master for 4.4 release.
Once merged, it shall be cherry-picked to 4.3.

@pliurh PTAL.

**Comment by pliurh** [comment]
LGTM

File: networking/multiple_networks/configuring-sr-iov.adoc [hunk 1]
```diff
@@ -9,39 +9,54 @@ toc::[]
...
 
 {product-title} includes the capability to use SR-IOV hardware on your nodes.
-You can attach SR-IOV virtual function (VF) interfaces to Pods on nodes with SR-IOV hardware.
+You can attach SR-IOV virtual function (VF) interfaces to Pods on nodes with
+SR-IOV hardware.
 
 You can use the {product-title} console to install SR-IOV by deploying the
 SR-IOV Network Operator. The SR-IOV Network Operator creates and manages the
-components of the SR-IOV stack. The Operator provides following features:
+components of the SR-IOV stack. The Operator provisions following components:
```
Comment (lamek) [inline networking/multiple_networks/configuring-sr-iov.adoc]: Minor- the article "the is missing:
"The Operator provisions the following components:"

File: networking/multiple_networks/configuring-sr-iov.adoc [hunk 2]
```diff
@@ -9,39 +9,54 @@ toc::[]
...
+* Provision the SR-IOV network config daemon on worker nodes.
+* Provision the Operator webhook on master nodes.
+* Provision the Network resources injector on master nodes.
+* Provision the SR-IOV network device plug-in on worker nodes.
+* Provision the SR-IOV CNI plug-in executable on worker nodes.
 
 Here's the function of each above mentioned SR-IOV components.
 
+* The SR-IOV Operator is a Kubernetes Deployment that manages all SR-IOV
+components in cluster. It watches creation, update and deletion of Operator
```
Comment (lamek) [inline networking/multiple_networks/configuring-sr-iov.adoc]: Missing the article "a":
"...components in a cluster."

File: networking/multiple_networks/configuring-sr-iov.adoc [hunk 3]
```diff
@@ -9,39 +9,54 @@ toc::[]
...
+Custom Resources and takes corresponding actions such as generating
+NetworkAttachmentDefinition Custom Resources for SR-IOV CNI, creating and
+updating configuration of SR-IOV network device plug-in etc.
```
Comment (lamek) [inline networking/multiple_networks/configuring-sr-iov.adoc]: Add a comma before "etc.":
"updating configuration of SR-IOV network device plug-in, etc."

**Review by lamek — COMMENTED** [review]
@zshi-redhat Some very minor nits, and otherwise LGTM.

Can you please update and squash so we have a single commit?

File: networking/multiple_networks/configuring-sr-iov.adoc (hunk 1 above)
Comment (zshi-redhat) [reply networking/multiple_networks/configuring-sr-iov.adoc]: Done

File: networking/multiple_networks/configuring-sr-iov.adoc (hunk 2 above)
Comment (zshi-redhat) [reply networking/multiple_networks/configuring-sr-iov.adoc]: Done

File: networking/multiple_networks/configuring-sr-iov.adoc (hunk 3 above)
Comment (zshi-redhat) [reply networking/multiple_networks/configuring-sr-iov.adoc]: Done

**Comment by zshi-redhat** [comment]
> @zshi-redhat Some very minor nits, and otherwise LGTM.
> 
> Can you please update and squash so we have a single commit?

Thanks @lamek for the review!
Updated and squashed into one commit.

**Pull request merged**